### ISSUE TITLE
Server-side batch changes Beta Phase

### ISSUE THREAD

**Issue by jyavorska** [sourcegraph/sourcegraph#30817]
## Problem to solve

For the Beta, unblock leading tech organizations.

- Unlock large enterprise Batch Changes use case.
- Allow for a smooth user flow, starting with a search (or another feature) and ending on a batch change

## Measure of success

- 3 customers create more than 200 changesets server-side
- 2 customers GA running batch changes server-side internally
- One customer agrees to quote / case study

## Solution summary

- Scalability (10,000 changeset scale)
- SSBC UI and editor is mature enough
- Feature is fully documented
- We have defined an upgrade policy and have a docs page for it

## Out of scope

- Managed executors are out of scope. Only self-hosted executors are inn scope

Artifacts:
- [PD 26 APPROVED: server side batch changes](https://docs.google.com/document/d/1MnfaA9P7mtur7aQbMvy2x96_zy_IrOhY-MbGUo5I5wk/edit#heading=h.g0gjwch98szj)

## What specific customers are we iterating on the problem and solution with?

- https://github.com/sourcegraph/accounts/issues/580
- https://github.com/sourcegraph/accounts/issues/565

## Impact on use cases

This makes using Batch Changes for code health, fixing security vulnerabilities, incident response, and code reuse much easier at scale. It's also a key part in enabling having a one-click batch change from a library

## Delivery plan

- [x] Deliver engineering milestone "EM3" tracked in #22764 
- [x] Release brief (target customers, Q&A)
- [x] PMM material: demo video, GIF, tweets @malomarrec  to sync with @danielmarquespt 
- [x] CE training session @malomarrec 
- [x] Train and handoff first-line support to customer support cc @chrispine

**Comment by jyavorska** [comment]
@malomarrec it looks like https://github.com/sourcegraph/sourcegraph/issues/27841 are duplicates now. You should merge them in some way so there's only one. I'd recommend keeping this one since it already has the new structure, but up to you.

**Comment by malomarrec** [comment]
- SSBC Beta has been shipped
- CE and AE training completed last week, documented here: https://handbook.sourcegraph.com/departments/engineering/dev/code-graph/batch-changes/go-to-market/
- Chris had been working on handing-off support. 
- Additional post-release items are tracked here: https://github.com/orgs/sourcegraph/projects/216/views/1
 
@chrispine can we consider this done and close this issue?

**Comment by malomarrec** [comment]
This is live! Woot woot! Congrats to everybody :)
